docs/spec/spec.md

```diff
@@ -0,0 +1,283 @@
+# Specification: Ballerina `aws.redshiftdata` Connector
+
+_Authors_: @ayeshLK \
+_Reviewers_: TBA \
+_Created_: 2025/02/27 \
+_Updated_: 2025/02/27 \
+_Edition_: Swan Lake 
+
+## Introduction
+
+This is the specification for the `aws.redshiftdata` connector of [Ballerina language](https://ballerina.io/), which provides the 
+functionality to execute database operations on an AWS Redshift cluster or an AWS Redshift Serverless work group using an HTTP API.
```

Comment:
Shall we omit the line breaks within paragraphs? Some Markdown renderers add a line break for new lines, which might cause incorrect formatting.

Reply:
Fixed with: [08420fa62774f5a81d18654e7e83b71f83d77be2](https://github.com/ballerina-platform/module-ballerinax-aws.redshiftdata/pull/26/commits/08420fa62774f5a81d18654e7e83b71f83d77be2)